framework/include/utils/MooseTypes.h

```diff
@@ -23,6 +23,10 @@
 #include "libmesh/elem.h"
 #include "libmesh/petsc_macro.h"
 #include "libmesh/boundary_info.h"
+#include "libmesh/parameters.h"
+
+// BOOST include
+#include "bitmask_operators.h"
```

Comment:
Do you remember what version of Boost this header originally came from? It seems very useful but I don't see it in Boost versions going back to 1.61.

Reply:
Don't know, but it's been in libMesh for awhile. I've been using it in the GrainTracker since early last year before we added BOOST to the package.

Reply:
OK - I'm senile. I added this to MOOSE